src/routes.js

```diff
@@ -1,23 +1,43 @@
-import Home from './pages/home.vue';
+// import Home from './pages/home.vue';     // <== Can be removed pending PR review
+import coreRoutes from 'tommy-core/src/routes';
 import Settings from './pages/settings.vue';
 import AddonDetails from './pages/addon-details.vue';
+import Addons from './pages/addons.vue';
 
-import coreRoutes from 'tommy-core/src/routes';
+import Views from './pages/views.vue';
+import config from '../config.json';
+
+let StartPage = undefined;
+if (config.starting_page) {
+  StartPage = () => import('./pages/' + config.starting_page + '.vue');
+} else {
+  StartPage = () => import('./pages/home.vue');
+}
 
 const routes = [
   {
     path: '/',
-    component: Home,
+    component: StartPage,
```

Comment:
Might be more elegant to do something like this?
`{ path: '/a', redirect: config.starting_page }`
Untested, but I'm guessing something similar should work...

Reply:
Yep, that works. 

Suggest line should be `redirect: { name: config.starting_page }` to use the named view and avoid having to handle trailing `/`'s in the config (so the config can still be something like `"starting_page" : "addons"`).